docs/am43xx.md

```diff
@@ -0,0 +1,57 @@
+# OP-TEE on Texas Instruments AM43xx
```

Comment:
Maybe it's better to have just one readme for all TI platforms. Build/deploy steps are the sam, the only difference is in SECDEV package to use (just to mention that we need different packages for different TI SoCs)